### PR TITLE
Added help to manpage how to add a remote machinery user

### LINKED ISSUE #1162: Document requirements for remote user
The man page specifies the required entries in the sudoers file. The requirements for the remote user should be also documented on the man page:
https://github.com/SUSE/machinery/blob/master/man/machinery-inspect.1.md 

See: http://lists.suse.com/pipermail/machinery/2015-July/000021.html

### PR DESCRIPTION
**Pull request by vwallfahrer**
Fixes #1162

### CODE REVIEW

File: man/machinery-inspect.1.md [hunk 1]
```diff
@@ -106,6 +106,14 @@ trigger errors.
 
     machinery ALL=(ALL) NOPASSWD: /usr/bin/find,/usr/bin/cat,/bin/cat,/usr/bin/rsync,/bin/rpm -V *,/bin/tar --create *
 
+  * To add a remote `machinery` user run as root:
+
+    # `useradd -m  machinery  -c "remote user for machinery"`
```
Comment (tgoettlicher) [inline man/machinery-inspect.1.md]: Are the double spaces on purpose?

Comment (vwallfahrer) [inline man/machinery-inspect.1.md]: No, I will fix that.

**Comment by tgoettlicher** [comment]
LGTM. Will you tell the requester that you fix the issue he reported?

**Comment by vwallfahrer** [comment]
Sure!

**Pull request merged**